app/assets/javascripts/wizard/addon/components/homepage-preview.js

```diff
@@ -361,7 +359,9 @@ export default createPreviewComponent(659, 320, {
 
   renderLatest(ctx, colors, font, width, height) {
     const rowHeight = height / 6.6;
-    const textColor = darkLightDiff(colors.primary, colors.secondary, 50, 50);
+    const textColor = colors.primary_medium // accounts for hard-set color variables in solarized themes
+      ? colors.primary_medium
+      : darkLightDiff(colors.primary, colors.secondary, 50, 50);
```

Comment:
```suggestion
    // accounts for hard-set color variables in solarized themes
    const textColor = colors.primary_medium || darkLightDiff(colors.primary, colors.secondary, 50, 50);
```

(you may need to run prettier)